tests/cases/fourslash/autoImportTypeImport1.ts

```diff
@@ -0,0 +1,41 @@
+/// <reference path="fourslash.ts" />
+
+// @verbatimModuleSyntax: true
+// @target: esnext
+
+// @Filename: /foo.ts
+//// export const A = 1;
+//// export type B = { x: number };
+//// export type C = 1;
+//// export class D = { y: string };
+
+// @Filename: /test.ts
+//// import { A, D, type C } from './foo';
+//// const b: B/**/ | C;
+//// console.log(A, D);
+
+goTo.marker("");
+
+verify.importFixAtPosition([
+`import { A, D, type C, type B } from './foo';

```

Comment:
Since it's adding `type B` and it's determined that the types are unsorted (since it specified `inline`), it adds it on to the end. If the imports were unsorted before, it would always add onto the end, otherwise it might add the import somewhere weird in the middle, since we're not changing the order of the rest of the imports